src/Control/Inciter/Options/Problem.h

```diff
@@ -26,6 +26,8 @@ enum class ProblemType : uint8_t { USER_DEFINED=0,
                                    SHEAR_DIFF,
                                    DIR_NEU,
                                    VORTICAL_FLOW,
+                                   NL_ENERGY_GROWTH,
```

Comment:
... and I greatly appreciate it! Please do not take this is a criticism of your code review. (As I do not take your code review as a negative criticism of the code I wrote but as an effort to improve code quality and catch errors.)